charts/kueue/values.yaml

```diff
@@ -19,6 +19,11 @@ controllerManager:
       tag: v0.8.0
       # This should be set to 'IfNotPresent' for released version
       pullPolicy: IfNotPresent
+    containerSecurityContext:
```

Comment:
if it can perfectly run in a restricted environment... let's not make it configurable?

Reply:
OK, that seems fine. I thought folks would want to make it even more restricted, (e.g. using `readOnlyRootFilesystem: true`, which I haven't bothered to do here, or a more restrictive `seccompProfile`). 

I'm happy to just change it and leave it unconfigurable too.

Off the top of your head, do you know which directory/ies `kueue` tries to write to? (so we can make them `emptyDir`s and restrict read/write of root FS). No worries if it's too complicated to just write, I asked in case it's easy for you.

Reply:
We don't write to disk at all and I don't think we intend to. kueue is pretty stateless.

Reply:
OK, I'll look into why `readOnlyRootFilesystem: true` failed on my cluster then.

Reply:
Uhmm... I think I remember seeing that the cert logic writes to disk.